firefox_puppeteer/tests/test_about_window.py

```diff
@@ -0,0 +1,63 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestAboutWindow(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.about_window = self.browser.open_about_window(trigger='menu')
+
+    def tearDown(self):
+        try:
+            self.windows.close_all([self.browser])
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def test_basic(self):
+        self.assertEqual(self.about_window.window_type, 'Browser:About')
+
+    def test_check_for_updates_and_download(self):
+        # testing both of these as they need to occur in this order
+        self.about_window.check_for_updates()
+        self.assertEqual('downloadAndInstall', self.about_window.wizard_state)
+        self.about_window.download()
+        self.assertEqual('apply', self.about_window.wizard_state)
```

Comment:
I can see from Travis that this fails because there are no updates available, so I think we need to just nix this entire test.